src/core/meshcop/commissioner.cpp

```diff
@@ -916,8 +964,9 @@ void Commissioner::SendJoinFinalizeResponse(const Coap::Header &aRequestHeader,
     mNetif.GetDtls().Send(buf, static_cast<uint16_t>(cur - buf));
     mSendKek = false;
 
-    otLogInfoMeshCoP("sent joiner finalize response\r\n");
-    otLogCertMeshCoP("[THCI] direction=send | type=JOIN_FIN.rsp\r\n");
+    otLogInfoMeshCoP("sent joiner finalize response\n");
+    otLogCertMeshCoP("[THCI] direction=send | type=JOIN_FIN.rsp\n");
```

Comment:
In addition, adding a `'r` for Windows logs causes an unnecessary extra line, which makes it look very ugly.